app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -52,6 +55,15 @@ class WallpaperSettingsFragment : Fragment() {
                     var coroutineScope = rememberCoroutineScope()
 
                     WallpaperSettings(
+                        onLearnMoreClick = {
+                            (activity as HomeActivity).openToBrowserAndLoad(
+                                searchTermOrURL = SupportUtils.getGenericSumoURLForTopic(
```

Comment:
I don't think we will want to convert this into a SUMO URL, as I believe we'll be given links to exact web pages. Even if we end up having a SUMO article for this particular collection, I think in the future it would be possible for this link to lead to a partner site, for example.